src/config.rs

```diff
@@ -1056,7 +1057,12 @@ impl Config {
             "BatchInstall" => self.batch_install = true,
             "UseAsk" => self.use_ask = true,
             "SaveChanges" => self.save_changes = true,
-            "NewsOnUpgrade" => self.news_on_upgrade = true,
```

Comment:
Seems weird to have 2 ways to set the news url. I think just the direct option is fine.

Reply:
Sorry I didn't get it. What do you mean by the direct option?

Reply:
Do you mean that we should remove `news_on_upgrade` and only leave `news_url`?

Reply:
yeah